Alueta_MarkRoland/Activity3/Resume/resume.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Activity 3 - Online Resume of Mark Roland C. Alueta - Advance</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <center>
+      <img src="./assets/me but white.jpg" />
+
+      <h1>Mark Roland C. Alueta</h1>
+      <h3>Former Special Science Teacher I</h3>
+
+      <div>
+        <span>+63 987 654 3210</span> |
+        <span> <a href="">markintoshroland@gmail.com</a> | </span>
```

Comment:
You can try to maximize this by using mailto
href="mailto:markintoshroland@gmail.com"